docker/development/newdle.cfg.docker

```diff
@@ -0,0 +1,92 @@
+# A random string that is secret.
+SECRET_KEY = 'jfAvAzpJRRfzk7ceanJzJpEIGqa3IzMs'
```

Comment:
i wouldn't put an actual value in there but generate one ;)

Reply:
well this is just for the sample docker-compose maybe a `secret` or `change-me` value would be more clear. Wdyt?